package.yaml

```diff
@@ -126,6 +126,7 @@ library:
     - singletons-base >= 3.0 && < 3.3
     - text >= 1.2 && < 2.1
     - nothunks >= 0.1 && < 0.4
+    - machines >=0.7 && <0.8
```

Comment:
looking at CI, it looks like we want a more strict interval here.

Looking at https://flora.pm/packages/@hackage/machines/0.7, it looks like that only `machines` 0.7.3 as tested with GHC 9.

Could you try bumping up the lower bound and see whether the CI passes?

Reply:
Let's try it!